tests/test_serializer_klc.py

```diff
@@ -134,6 +133,32 @@ def test_intl_keymap():
         39	SPACE	0	0020	0020	-1	-1	//
         """
     )
+    assert len(keymap) == len(keymap_ref)
+    assert keymap == keymap_ref
+
+    extraMapping = {
+        # NOTE: redefine level
+        "ae01": {"shift": "?"},
+        # TODO
+        # # NOTE: test case variants and ODK alias
+        # "kppt": {"base": ",", "sHiFt": ";", "1dk": ".", "ODk_shiFt": ":"},
+        # NOTE: clone level
+        "esc": {"base": "\x1b", "shift": "(ae11)"},
+        # NOTE: clone key
+        "henk": "(lsgt)",
+    }
```

Comment:
I assume this is a work in progress. Otherwise the comments could be more explicit, I’m not following you here. :bow:

Reply:
Fixed